docs/docsite/rst/dev_guide/style_guide/spelling_word_choice.rst

```diff
@@ -103,7 +103,7 @@ Fewer is used with plural nouns. Think things you could count.  Time, money, dis
 
 File name
 ^^^^^^^^^
-Correct. Do not use "filename."
+Correct. Do not use "file name."
```

Comment:
Remove this change.  this is where we actually tell people not to use filename :-)